docker/Dockerfile_Process

```diff
@@ -20,7 +21,13 @@ RUN cd home \
 
 RUN cd home \
     && pwd \
-    && ./install_ant-media-server.sh -i ${AntMediaServer} -s false
+    && if [ -n "$AntMediaServer" ]; then \
+           ./install_ant-media-server.sh -i ${AntMediaServer} -s false; \
+       fi \
+    && if [ -n "$LicenseKey" ]; then \
+           ./install_ant-media-server.sh -l ${LicenseKey} -s false; \
+       fi
+
```

Comment:
What if there is no {AntMediaServer} and {LicenseKey}?

Reply:
Let's do that by adding an exit code.